charts/t8s-cluster/templates/management-cluster/clusterClass/k0smotronControlPlaneTemplate/_k0smotronControlPlaneTemplateSpec.yaml

```diff
@@ -50,4 +53,30 @@ k0sConfig:
 etcd:
   persistence:
     size: 5Gi
+{{/*rolloutBefore:*/}}
```

Comment:
Could you explain whats going on here?

Reply:
Ah. this is how it's done with the kubeadmControlPlane.

Can this be removed then? As the option is now implemented